translations/tiled_ru.ts

```diff
@@ -3265,12 +3265,12 @@ Please select specific format.</source>
     <message>
         <location line="+4"/>
         <source>Group Layer</source>
-        <translation>Сгруппировать слой</translation>
+        <translation>Группа слоев</translation>
```

Comment:
Hard to tell what is 'group' there - a noun or a verb, because in russian that will make difference. You could look it in the redactor in menu `Layer->New` and `Layer->Group`.
`Layer->New->Group Layer` should be a noun, but `Layer->Group->Group Layer` a verb.

Reply:
So based on functional of these menu options I assume that for Russian translation should be two different strings for each menu. It will be an equivalent for English 'Layer->New->a Layer of Group' and 'Layer->Group->perform Group to Layer'.
Maybe @bjorn could clarify what exactly the context is.

Reply:
Unfortunately both "AddGroupLayer" and "GroupLayers" actions have the same context and the same text, so it is currently not possible to have different translations for them. I'll fix that for Tiled 1.1 by using a disambiguation string. For this PR, just pick one.

Reply:
@bjorn So it doesn't matter actually, let it be the updated string. One menu option will be ok and other...confusing =)